ZipBuilder/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -32,21 +33,24 @@ enum Architecture: String, CaseIterable {
       case .simulator:
         // No extra arguments are required for simulator builds.
         return []
+      case .catalyst:
+        return []
       }
     }
   }
 
   case arm64
-  case arm64e
-  case armv7
-  case i386
+  //case armv7
+  //case i386
   case x86_64
+  case x86_64h // x86_64h, Haswell, used for Mac Catalyst
 
   /// The platform associated with the architecture.
   var platform: TargetPlatform {
     switch self {
-    case .arm64, .arm64e, .armv7: return .device
-    case .i386, .x86_64: return .simulator
+    case .arm64: return .device
```

Comment:
We'll need to keep 32 bit slices a bit longer. Another way to disable is using the [archs option ](https://github.com/firebase/firebase-ios-sdk/blob/master/ZipBuilder/Sources/ZipBuilder/LaunchArgs.swift#L61)

Reply:
Fixed it!